src/averageAges.js

```diff
@@ -55,7 +69,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const children = onlyWithSon
+    ? people.filter(p => p.sex === 'm'
+      && people.some(m => m.name === p.mother))
+    : people.filter(p => people.some(m => m.name
+      === p.mother));
```

Comment:
Fix formatting according to the [checklist](https://github.com/mate-academy/js_average-ages_test-task/blob/master/checklist.md).
```suggestion
  const children = onlyWithSon
    ? people.filter(
        p => p.sex === 'm' && people.some(m => m.name === p.mother)
      )
    : people.filter(p => people.some(m => m.name === p.mother));
```